webapp/components/wpt-flags.html

```diff
@@ -0,0 +1,71 @@
+<!--
+Copyright 2018 The WPT Dashboard Project. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file.
+-->
+
+<link rel="import" href="../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../bower_components/paper-item/paper-item.html">
+<link rel="import" href="../bower_components/paper-checkbox/paper-checkbox.html">
+
+<!--
+`<wpt-flags>` is a component for checking wpt.fyi feature flags.
+-->
+<dom-module id="wpt-flags">
+  <script>
+    const WPT_FYI_FEATURES = ['queryBuilder'];
+
+    const _wptFlags = (superClass, opt_writable) => class extends superClass {
```

Comment:
The name `opt_writable` is a bit unclear to me. What is `opt`? Optional?

Also, what about using `readOnly` directly as the argument to avoid the extra negation?

Reply:
Done.